### PR TITLE
Features 

### PR DESCRIPTION
**Pull request by zied2112**
Adding New Web pages

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Header 1</h1>
```
Comment (Nemwel-Boniface) [inline index.html]: - [ ] Kindly note that according to the project requirements, the ```<h1>``` should have a class to style your h1.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+h1{
+    background-color: antiquewhite;
+}
```
Comment (Nemwel-Boniface) [inline style.css]: - [ ] Kindly use the CSS class selector for styling this ```<h1>```.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (Nemwel-Boniface) [inline .gitignore]: - [ ] Kindly set up the project linters. Currently, I can only see the test.md file ignored but not the node_modules/.

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @zied2112            :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

You have done an amazing job with setting up your Hello Microverse project :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

- [ ] Kindly set the project linters so that you meet all the project requirements. You can see them by following [this](https://github.com/microverseinc/linters-config) link.
  - Then follow the first one labeled "[html&css](https://github.com/microverseinc/linters-config/blob/master/html-css)" and follow the steps provided there.

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zied2112** [comment]
I don't know why the Stylint is making errors ?
image

**Comment by Nemwel-Boniface** [comment]
Hello @zied2112 . To auto fix any linters run ```npx stylelint "**/*.{css,scss}" --fix```. But following the linter errors, that should be written like this:

```
.header {
  background-color: antiquewhite;
}
```

**Comment by zied2112** [comment]
> Hello @zied2112 . To auto fix any linters run `npx stylelint "**/*.{css,scss}" --fix`. But following the linter errors, that should be written like this:
> 
> ```
> .header {
>   background-color: antiquewhite;
> }
> ```

I think merging and deleting this branch is not included in this project no?

**Comment by Nemwel-Boniface** [comment]
@zied2112   You should only merge it when it is approved by a code reviewer. Deleting that branch once you merge it is a personal choice.

**Comment by zied2112** [comment]
and you are not the one? should I request a review?

**Comment by Nemwel-Boniface** [comment]
No @zied2112  I may not be the one to review it again. When you have made the requested changes you are supposed to resubmit the link of the PR in your dashboard. Do so ASAP!

**Review by Reem-lab — COMMENTED** [review]
Hi @zied2112, 👋🏻👋🏻 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that you submit a review in the wrong activity so you should submit in the hello microverse activity in your dashoard since you are submitting it in the Portfolio first setup activity 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by zied2112** [comment]
> Hi @zied2112, 👋🏻👋🏻
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> You have submitted a project, that you submit a review in the wrong activity so you should submit in the hello microverse activity in your dashoard since you are submitting it in the Portfolio first setup activity
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

Yeah, I made a fault when I was submitting the activity. Sorry for this

**Review by phelian23 — APPROVED** [review]
Hi @zied2112,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**